user-interface/src/case-detail/panels/CaseDetailHeader.tsx

```diff
@@ -1,10 +1,16 @@
 import './CaseDetailHeader.scss';
-import { Link } from 'react-router-dom';
-import Icon from '@/lib/components/uswds/Icon';
 import { getCaseNumber } from '@/lib/utils/formatCaseNumber';
 import { useEffect } from 'react';
 import useFixedPosition from '@/lib/hooks/UseFixedPosition';
 import { CaseDetail } from '@common/cams/cases';
+import Icon from '@/lib/components/uswds/Icon';
+
+export function copyCaseNumber(caseId: string | undefined): void {
```

Comment:
I think this would be best in lib/utils next to the getCaseNumber function, as opposed to on the CaseDetailHeader for reusability purposes.